src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`3.cs

```diff
@@ -139,28 +139,19 @@ protected override void SetupNewTextView(IVsTextView textView)
                     ProducerPopulatedTagSource<IOutliningRegionTag> tagSource;
                     if (outliningTaggerProvider.TryRetrieveTagSource(wpfTextView, wpfTextView.TextBuffer, out tagSource))
```

Comment:
Not necessary to get the tag source anymore. Was TryRetrieveTagSource only called here and in the test? Can more be deleted?